src/cljs/athens/views/blocks.cljs

```diff
@@ -834,8 +841,13 @@
                         :caret-position    nil
                         :linked-ref/open (or (false? linked-ref) initial-open)})]
 
-     (fn [block linked-ref-data]
+     (fn [block linked-ref-data opts]
        (let [{:block/keys [uid string open children _refs]} block
+             uid-sanitized-block (s/transform
+                                   (specter-recursive-path #(contains? % :block/uid))
+                                   (fn [{:block/keys [original-uid uid] :as block}]
+                                     (assoc block :block/uid (or original-uid uid)))
+                                   block)
```

Comment:
Reminder to learn this

Reply:
I love specter for this exact reason. It's almost impossible to write this code in plain clj(s) -- and it will be very hard to read.